.github/workflows/release.yml

```
@@ -32,18 +32,49 @@ jobs:
         uses: actions/setup-go@v6
         with:
           go-version: '1.25'
+          cache: false # Disable cache - we clean between regions to save disk
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)